<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
  </property>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400;&quot;&gt;hh:mm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="helloLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>2</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;helloLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inboxBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>收件箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="workBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>办公</string>
   </property>
  </widget>
  <widget class="QPushButton" name="friendBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contactBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>草稿箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outboxBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>发件箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emailBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>垃圾</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>标记</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;
background-color: rgb(248, 246, 221);
border:1px solid lightgray;
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑 Light'; font-size:9pt; font-weight:24; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>暂无内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:1px solid lightgray; } 
QPushButton:hover{border-color:green;}</string>
   </property>
   <property name="text">
    <string>写信</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classifyComboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{background-color: palegoldenrod;border-width: 1px;
border-color: darkkhaki;border-style: solid;border-radius: 5;} 
QComboBox:hover{border-color:green;}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:0px solid lightgray; background-color:transparent} 
QPushButton:hover{color:green;}</string>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list2Play">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;
border:1px solid lightgray;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <item>
    <property name="text">
     <string>邮件</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Networklabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
